x/bank/simulation/operations_test.go

```diff
@@ -137,6 +197,23 @@ func (suite *SimTestSuite) getTestingAccounts(r *rand.Rand, n int) []simtypes.Ac
 	return accounts
 }
 
+func (suite *SimTestSuite) getModuleAccounts(n int) []authtypes.ModuleAccountI {
+	accounts := make([]authtypes.ModuleAccountI, n)
+	amount := sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(1)))
+
+	for i := 0; i < n; i++ {
+
+		// what method  is appropriate to use here? maybe NewEmptyModuleAccount
+		macc := suite.app.DistrKeeper.GetDistributionAccount(suite.ctx)
```

Comment:
Is GetDistributionAccount appropriate method to generate Module Account or i have to use NewEmptyModuleAccount?

Reply:
I think we should use `accountKeeper.GetModuleAddress(ModuleName)` to get module accounts. Also, we don't need any loop here. We can directly return all the group accounts available.